### PR TITLE
Don't used untyped modules

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Removes usage in the tests of modules that are not typed, which should not be considered public.

Refs https://github.com/libero/article-store/pull/29#issuecomment-572475448.

### CODE REVIEW

File: test/router.test.ts [hunk 1]
```diff
@@ -7,7 +6,7 @@ describe('router', (): void => {
 
   Object.keys(Routes).forEach((name: string): void => {
     it(`has a route named ${name}`, (): void => {
-      expect(router.route(Routes[name])).toBeInstanceOf(Layer);
+      expect(router.route(Routes[name])).toBeTruthy();
```
Comment (thewilkybarkid) [inline test/router.test.ts]: No way to access the object unfortunately. `route()` returns a `Layer` or `false`.

**Review by giorgiosironi — APPROVED** [review]
Though dropping them because they are private, more than because of the lack of typing?

File: test/router.test.ts (hunk 1 above)
Comment (giorgiosironi) [reply test/router.test.ts]: to be an object (stronger than being truthy)?

Comment (giorgiosironi) [reply test/router.test.ts]: (no idea of the matchers available here)

**Comment by thewilkybarkid** [comment]
More because of the latter, but wasn't exactly happen with using them.

File: test/router.test.ts (hunk 1 above)
Comment (thewilkybarkid) [reply test/router.test.ts]: https://github.com/libero/article-store/pull/165/commits/92f126a3363e636917734b11238e2d190cca4969 is the best I can think of that still gives a useful message.

File: test/router.test.ts [hunk 2]
```diff
@@ -7,7 +6,7 @@ describe('router', (): void => {
 
   Object.values(Routes).forEach((name: string): void => {
     it(`has a route named ${name}`, (): void => {
-      expect(router.route(name)).toBeInstanceOf(Layer);
+      expect(router.route(name)).toMatchObject({ name });
```
Comment (giorgiosironi) [inline test/router.test.ts:9]: even better than a simple `object` type

**Pull request merged**